shared/deployment/secrets/_default.md

```diff
@@ -72,12 +72,22 @@ build-secrets:
                dest: my-recipe2
 ```
 
-This will mount the `super-secret-recipe` file as `/run/secrets/my-recipe` for all services, and `super-secret-recipe-2` as `/run/secrets/my-recipe2` for `service1` only. Again, note that the `/run/secrets` folder is only available during the image build, and not present in the image that is deployed to the devices.
+This will mount the `super-secret-recipe` file into `/run/secrets/my-recipe` file in
+every build container. However, the `/run/secrets` folder will not be added to the
+"build context" (tar stream) that is sent to the Docker daemon (or balenaEngine),
```

Comment:
```suggestion
build context that is sent to the Docker daemon (or balenaEngine),
```

I'd remove quotes as build context is consistent with terminology we have used elsewhere.